src/zmq/zmqnotificationinterface.h

```diff
@@ -8,6 +8,7 @@
 #include "validationinterface.h"
 #include <string>
 #include <map>
+#include <list>
```

Comment:
Ah, I was viewing the diff and didn't realize it was truncated, so I didn't see the list :) ACK the include.